install.py

```
@@ -176,21 +175,21 @@ def main(argv: list[str] | None = None) -> int:
     print(f"- commands: {install_dir / 'commands'}")
     print(f"- agents:   {install_dir / 'agents'}")
     print(f"- skills:   {install_dir / 'skills'}")
-    print(f"- prompts:  {install_dir / 'codeagent'} (*-prompt.md placeholders)")
+    print(f"- prompts:  {install_dir / 'fish-agent-wrapper'} (*-prompt.md placeholders)")
     if wrapper_path is not None:
         print(f"- wrapper:  {wrapper_path} (copied from ./dist)")
 
     if str(install_dir) != str(Path(DEFAULT_INSTALL_DIR).expanduser().resolve()):
         print("")
         print("Note:")
         print(f"- You used a non-default install dir.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)